### DISCUSSION TITLE
Is it possible to align NodeJS version with AWS Lambda NodeJS version?

### DISCUSSION THREAD

**Discussion by tal-rofe** [renovatebot/renovate#27501] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I want my project to align with AWS Lambda NodeJS version.
In AWS console, I get the version:

<img width="183" alt="image" src="https://github.com/renovatebot/renovate/assets/100444463/6b03a7f2-ee3e-42a1-80f5-5eff7d6f120e">

That said, I'd like to align my project NodeJS's version to AWS' version.

Of course, I won't expect AWS to align its version to most up-to-date LTS version of NodeJS.

So, I have 2 options:
1. use container image in AWS to manage my own NodeJS version to be LTS most up to date
2. use other source for NodeJS versions (not the one Renovate uses by default) - to match AWS version

So my question is, regarding option 2, is there a source for such thing to apply with Renovate? so that Renovate always pushes pull requests to match AWS versions?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
[registryUrls ](https://docs.renovatebot.com/configuration-options/#registryurls), would be the option to go. But for the `node-version` datasource, which is used to fetch node versions, the `customRegistryUrl` support is still not implemented.
